### ISSUE TITLE
"conda update --all" fail after upgrading to conda 4.4.2

### ISSUE THREAD

**Issue by lijunzh** [conda/conda#6532]
Below is the output:

```
$ conda update --all
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - anaconda[version='>1']

Current channels:

  - https://repo.continuum.io/pkgs/main/osx-64
  - https://repo.continuum.io/pkgs/main/noarch
  - https://repo.continuum.io/pkgs/free/osx-64
  - https://repo.continuum.io/pkgs/free/noarch
  - https://repo.continuum.io/pkgs/r/osx-64
  - https://repo.continuum.io/pkgs/r/noarch
  - https://repo.continuum.io/pkgs/pro/osx-64
  - https://repo.continuum.io/pkgs/pro/noarch
```

I did a clean install of anaconda and re-try after upgrading conda and got the same results.

**Comment by kalefranz** [comment]
fixed with #6539

**Comment by lijunzh** [comment]
@kalefranz Thanks a lot for the quick response.

After update the conda by the following command:

```
$ conda update conda
Solving environment: done

## Package Plan ##

  environment location: /Users/lijun/anaconda3

  added / updated specs: 
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.4.3                |           py36_0         613 KB

The following packages will be UPDATED:

    conda: 4.4.2-py36_0 --> 4.4.3-py36_0

Proceed ([y]/n)? y


Downloading and Extracting Packages
conda 4.4.3: ########################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
```

Here is the current output:

```
$ conda update --all
Solving environment: done

# All requested packages already installed.
```

**Comment by drhay53** [comment]
I have this same problem. Anything that upgrades conda to 4.4.6 breaks everything; either packages cannot be found from any channel, or, if I try to update them, even individually, conda responds that they're already installed. The only thing that fixes things is a miniconda -bfp install run.

**Comment by kalefranz** [comment]
@drhay53 can you open a new issue and give more specifics? Give me whatever details needed to reproduce what you’re seeing. We just released 4.4.6, but we’ll pull it to limit the people affected if needed. If we pull 4.4.6, updates will fall back to 4.4.4.
